### ISSUE TITLE
Feature Requests

### ISSUE THREAD

**Issue by Zuescho** [devfake/flox#94]
Hi, first up i just wanted to let you know that this project looks really promising.
I really like the idea is there any chance that i version 2 you would add some function that you could add custom columns?

I'm searching for a nice way to keep an overview of my collection but at the same time keep it self hosted so that i do not have to rely on other services and could still add info manually like in my [Spreadsheet](https://docs.google.com/spreadsheets/d/1jDsnEhRQ5YkTI99w9snzhZ8m8EpVBr9r_p3q8vqiL3s/edit#gid=1002642137) right now.

So basically would you consider it possible to:
Add custom columns to Shows/Movies
Download the Covers when exporting (if TMDB ever shuts down its API )

With best regards.

**Comment by devfake** [comment]
Hey Zuescho,

thank you for your feedback and ideas.

We are currently working on some ideas that might be useful for you. Like `collections` to make a better seperations of your items. Or to label an item if you have a DVD/Blueray of them. Download the assets on export will definitely added.

We are not sure if we want to introduce customfields, but we have written it down as a feature.

**Comment by Zuescho** [comment]
Thanks for the good news ^^
Yeah i would love some sort of label, because i have most of my stuff seperated over 2 8TB Archive Drives 
Something like a BlueRay lable or something over the cover would be great!
Even greater if we could define some sort of lable ourself with some restrictions like only 5 characters so it would fit in some template.

Really looking forward to the 2. Version.

**Comment by devfake** [comment]
Yeah, we think a label system where the user can add his own tags would be great :)

Thank you again!
